packages/flutter_test/lib/src/widget_tester.dart

```diff
@@ -228,6 +231,20 @@ class WidgetTester extends WidgetController implements HitTestDispatcher, Ticker
     });
   }
 
+  /// See [AutomatedTestWidgetsFlutterBinding.addTime]
+  ///
+  /// This is the shorthand for
+  /// ```dart
+  ///     final AutomatedTestWidgetsFlutterBinding binding = tester.binding;
+  ///     binding.addTime(const Duration(seconds: ));
+  /// ```
+  void addTime(Duration duration) {
```

Comment:
I think in https://github.com/flutter/flutter/issues/19175#issuecomment-403607807 it was suggested to add this to `TestWidgetsFlutterBinding` instead as an empty implementation and then add an `@override` annotation to the existing impl in `AutomatedTestWidgetsFlutterBinding`.

Reply:
The `TestWidgetsFlutterBinding` has some other timeout related stuff as well (e.g. `defaultTestTimeout`). To keep everything at the same place, `addTime` should go here as well. Plus, that doesn't make the call side any more complicated. Both of your examples are almost equally concise. 

Additionally, moving it over there avoids the awkward type check you currently have in the implementation.

Reply:
Frankly, I am not totally convinced that "examples are almost equally concise", and do not necessarily agree that timeout related stuff should be in `TestWidgetsFlutterBinding`. However, a compromise right now is better than nothing, so I will continue anyway.

(The discussion on the conciseness/aesthetic of the sample code can continue later)